APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.m

```diff
@@ -81,6 +92,44 @@ - (void)signUpWithDataGroups:(NSArray<NSString *> *)dataGroups onCompletion:(voi
     }
 }
 
+- (void) signUpWithDataGroups:(NSArray<NSString *> *)dataGroups
+         withTestUserPromptVc:(__weak UIViewController*)vc
+                 onCompletion:(void (^)(NSError *))completionBlock
+{
+    [self signUpWithDataGroups:dataGroups
+          withTestUserPromptVc:vc
+              includeTestCheck:shouldPerformTestUserEmailCheckOnSignup
+                verifiedTester:NO
+                  onCompletion:completionBlock];
+}
+
+- (void) signUpWithDataGroups:(NSArray<NSString *> *)dataGroups
+         withTestUserPromptVc:(__weak UIViewController*)vc
+             includeTestCheck:(BOOL)includeTestCheck
+               verifiedTester:(BOOL)userVerifiedAsTester
+                 onCompletion:(void (^)(NSError *))completionBlock
+{
+    if (!includeTestCheck ||
+        ![[self.email lowercaseString] containsString:kHiddenTestEmailString])
+    {
+        [self signUpWithDataGroups:dataGroups onCompletion:completionBlock];
+        return;
+    }
+    
+    if (!userVerifiedAsTester)
+    {
+        [self showTestUserVerificationAlertWithDataGroups:dataGroups
+                                     withTestUserPromptVc:vc
+                                             onCompletion:completionBlock];
```

Comment:
I understand what you're saying, I like that idea.  Let me change it real quick.